chemistry/amber/amberformat.py

```diff
@@ -814,15 +834,24 @@ def write_parm(self, name):
             for i in xrange(len(self.parm_data[self.charge_flag])):
                 self.parm_data[self.charge_flag][i] *= CHARGE_SCALE
 
+        if self.vmd_compat is not None:
+            self.delete_flag('CTITLE')
```

Comment:
This should be moved into `convert_to_vmd_compat` -- `write_parm` shouldn't modify the parm object (permanently at least).